Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -134,6 +138,52 @@ private void OnSizeChanged(object sender, SizeChangedEventArgs e)
             }
         }
 
+        private void OnLoaded(object sender, RoutedEventArgs e)
+        {
+            _isLoaded = true;
+            DetermineOneRowMode();
+        }
+
+        private void OnUnloaded(object sender, RoutedEventArgs e)
+        {
+            _isLoaded = false;
+        }
+
+        private void DetermineOneRowMode()
+        {
+            if (_isLoaded)
+            {
+                var itemsWrapGridPanel = ItemsPanelRoot as ItemsWrapGrid;
+
+                if (OneRowModeEnabled)
+                {
+                    var b = new Binding()
+                    {
+                        Source = this,
+                        Path = new PropertyPath("ItemHeight")
+                    };
+
+                    itemsWrapGridPanel.Orientation = Orientation.Vertical;
```

Comment:
We should check for null on this, if ItemsPanelRoot happens to be something that's not ItemsWrapGrid, this will cause this line to throw an exception.